pkg/apis/workspaces/v1alpha2/doc.go

```diff
@@ -2,4 +2,5 @@
 // +k8s:deepcopy-gen=package,register
 // +k8s:openapi-gen=true
 // +groupName=workspace.devfile.io
+// +devfile:jsonschema:version=2.0.0-alpha2
```

Comment:
Yeah, hopefully this PR would help bootstrapping the version discussion.
However let me point out that this PR doesn't impose much constraint on versioning: It just assumes that K8S API versions would evolves more slowly than Json schema versions, which was already agreed on the last time we discussed about it, iirc. In other words, several Json Schema versions could correspond to a single K8S API.
However this lets us free to opt for a 1:1 mapping between KS API version and Json schema version, even though I personnally think it might be hard to manage over time.

Reply:
:+1: discussion can be elsewhere.